src/silkaj.py

```diff
@@ -1,51 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
It works for me (well I have another error, but it should be easy to fix). Do you have python installed?